draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -621,8 +621,14 @@ Other specifications or profiles of this specification may define alternative fo
 
 Security considerations in this section help achieve the following properties:
 
-* Claim values issued by the Issuer cannot be obtained by the adversary, if the Holder did not intend to disclose them.
-* Claim values that are not issued by the Issuer cannot be successfully verified by the Verifier, if the Holder forged the Disclosures.
+**Selective Disclosure:** An adversary in the role of the Verifier cannot obtain
+information from an SD-JWT about any claim name or claim value that was not
+explicitly disclosed by the Holder unless that information can be derived from
+other disclosed claims or external sources.
+
+**Integrity:** A malicious Holder cannot modify names or values of selectively disclosable claims without detection by the Verifier.
+
+Additionally, Holder Binding can be achieved as described in (#holder_binding_security).
```

Comment:
Maybe we should use the same format as for the other two properties. Re-using text from Brian's proposal:

```suggestion
**Holder Binding:** When Holder Binding is enforced, only the legitimate Holder of the credential can present the SD-JWT (as described in (#holder_binding_security)).
```

Reply:
I would be ok with 
> **Holder Binding:** Only the legitimate Holder of the credential can present the SD-JWT (as described in (#holder_binding_security)).